artifactory/cli.go

```diff
@@ -1042,7 +1043,18 @@ func mvnCmd(c *cli.Context) error {
 	if err != nil {
 		return err
 	}
-	mvnCmd := mvn.NewMvnCommand().SetConfiguration(buildConfiguration).SetConfigPath(configFilePath).SetGoals(filteredMavenArgs).SetThreads(threads).SetInsecureTls(insecureTls).SetDetailedSummary(detailedSummary).SetXrayScan(xrayScan)
+	filteredMavenArgs, format, err := coreutils.ExtractXrayOutputFormat(filteredMavenArgs)
+	if err != nil {
+		return err
+	}
+	if xrayScan && format != "" {
```

Comment:
Is this correct? Shouldn't it be - 
```
if !xrayScan && format != "" {
```
meaning - if scan was not requested and format was set?

The same question applies to the ```gradleCmd``` function.